Userland/Libraries/LibWeb/CSS/Parser/Parser.cpp

```diff
@@ -5537,19 +5537,29 @@ RefPtr<StyleValue> Parser::parse_grid_track_placement(Vector<ComponentValue> con
     if (!tokens.has_next_token()) {
         if (current_token.to_string() == "auto"sv)
             return GridTrackPlacementStyleValue::create(CSS::GridTrackPlacement());
+        // https://drafts.csswg.org/css-grid/#grid-placement-span-int
+        // If the <integer> is omitted, it defaults to 1.
+        if (current_token.to_string() == "span"sv)
```

Comment:
I don't think you want to be doing `.to_string()` here. Doing `.to_string()` is doing a bunch of extra stuff and allocating a new `AK::String`.  You can just do `current_token.ident()` to get a `StringView` you can compare against. I.e. this code should probably be: 
```c++
if (current_token.ident() == "span"sv) 
```
(There's more than just here where this can be changed, some from you're previous PRs that I can't review here).

Reply:
Thanks, I'll keep this in mind. Found a to_string() I had done in the rect parser as well :^)

Reply:
Btw I'm slightly wrong in my comment there it should be `current_token.is(Token::Type::Ident) && token.token().ident() == "whatever"sv`. You always have to check the token is the thing you expect first.

Reply:
Thanks! I'm doing equals_ignoring_case as well.